### ISSUE TITLE
Include all fields from sheet into item database

### ISSUE THREAD

**Issue by ghost** [computermuseumarnhem/cma_inventoryapp#9]
The sheet has many more columns than are shown in the items now, add all the fields.

**Comment by vandenzel** [comment]
Correct. Not all fields are in the item table (yet). I'm still thinking if I want a field for every column, to create a tag/value pair table that links to the item table.
